src/api-v1.js

```diff
@@ -665,13 +666,17 @@ api.declare({
       token: token,
     });
 
+    let workerId = slugid.v4();
+    let workerGroup = 'ec2_' + secret.region;
```

Comment:
Actually maybe it's better we don't do that in this PR... Probably we should limit number of retires allowed per token...